CMakeSettings.json

```diff
@@ -1,38 +1,42 @@
-{
-   // See https://go.microsoft.com//fwlink//?linkid=834763 for more information about this file.
-   "configurations": [
-      {
-         "name": "x64-RelWithDebInfo",
-         "generator": "Visual Studio 15 2017 Win64",
-         "configurationType": "RelWithDebInfo",
-         "inheritEnvironments": [ "msvc_x64_x64" ],
-         "buildRoot": "${projectDir}\\msvc.build_x64", // "${env.USERPROFILE}\\CMakeBuilds\\${workspaceHash}\\build\\${name}", 
-         "installRoot": "${projectDir}\\msvc.install_x64", // "${env.USERPROFILE}\\CMakeBuilds\\${workspaceHash}\\install\\${name}",
-         "cmakeCommandArgs": "-Wno-deprecated",
-         "variables": [
-            {
-               "name": "CMAKE_BUILD_TYPE",
-               "value": "RELWITHDEBINFO"
-            },
-            {
-               "name": "BUILD_FOR_WINSTORE",
-               "value": "OFF"
-            },
-            {
-               "name": "BUILD_64",
-               "value": "ON"
-            },
-            {
-               "name": "CMAKE_PREFIX_PATH",
-               "value": "${env.PATH};${projectDir}\\dependencies"
-            },
-            {
-               "name": "CMAKE_LIBRARY_PATH",
-               "value": "${projectDir}\\dependencies\\libx64"
-            }
-         ],
-         "buildCommandArgs": "-v",
-         "ctestCommandArgs": ""
-      }
-   ]
+﻿{
+  "configurations": [
+    {
+      "name": "x64-RelWithDebInfo",
+      "generator": "Visual Studio 16 2019",
+      "configurationType": "RelWithDebInfo",
+      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "buildRoot": "${projectDir}\\msvc.build_x64",
+      "installRoot": "${projectDir}\\msvc.install_x64",
+      "cmakeCommandArgs": "-Wno-deprecated",
+      "variables": [
+        {
+          "name": "CMAKE_BUILD_TYPE",
+          "value": "RELWITHDEBINFO",
+          "type": "STRING"
+        },
+        {
+          "name": "BUILD_FOR_WINSTORE",
+          "value": "OFF",
+          "type": "STRING"
+        },
+        {
+          "name": "BUILD_64",
+          "value": "ON",
+          "type": "STRING"
+        },
+        {
+          "name": "CMAKE_PREFIX_PATH",
+          "value": "${env.PATH};${projectDir}\\dependencies",
+          "type": "STRING"
+        },
+        {
+          "name": "CMAKE_LIBRARY_PATH",
+          "value": "${projectDir}\\dependencies\\libx64",
+          "type": "STRING"
+        }
+      ],
+      "buildCommandArgs": "-v",
+      "ctestCommandArgs": ""
+    }
+  ]
```

Comment:
Are you sure changes to this file won't break other people's workflows? This isn't a VSCode-specific file.

Reply:
most of these were automatically corrected by Code. 
The only thing I changed here was `"generator": "Visual Studio 15 2017 Win64"` that became `"generator": "Visual Studio 16 2019"`